python/tvm/script/_parser/core/dispatch.py

```diff
@@ -0,0 +1,156 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Parser dispatching infrastructure"""
+
+from typing import TYPE_CHECKING, Any, Callable, Dict, Optional, Tuple, Type
+
+from .doc import AST
+
+if TYPE_CHECKING:
+    from .parser import Parser
+
+
+ParseMethod = Callable[["Parser", AST], None]
+ParseVTable: Dict[Tuple[str, str], ParseMethod] = {}
+
+OpMethod = Callable[..., Any]
+OpVTable: Dict[Tuple[Type, AST, int], OpMethod] = {}
+
+
+def register(token: str, type_name: str):
+    """Register a method for a dispatch token and type name.
+
+    Parameters
+    ----------
+    token : str
+        The token for IR, e.g., T for TIR and R for Relax.
```

Comment:
Hi @MasterJH5574 - it seems this PR accidentally were merged some references to Relax in `main` as opposed to `unity` - can you push a follow-up PR and change it to the appropriate branch?

Reply:
I feel in this case it is a comment to talk about related references and is a minor thing that is fine as it is.

Reply:
I think the comment makes a lot of assumptions and is misleading, especially for new contributors that may or may not be expecting this reference to be here in this branch, so I'm still in favour of having this fixed.

As it might be situation in which the contribution is expected to be in another branch, rather than pushing myself a patch to fix the comment, I think it is best that the original contributor to push a synchronised that would fix here and in `unity`.